_neural-search-plugin/index.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: default
+title: Neural Search plugin
+nav_order: 1
+has_children: false
+has_toc: false
+---
+
+The Neural Search plugin is an experimental feature. For updates on the progress of Compare Search Results and other search relevance features, or if you want to leave feedback that could help improve the feature, make an issue in the [Neural Search repo](https://github.com/opensearch-project/neural-search).    
```

Comment:
Could we also link to this forum post for feedback: https://forum.opensearch.org/t/feedback-neural-search-plugin-experimental-release/11501